lte/gateway/python/scripts/pipelined_cli.py

```diff
@@ -51,8 +51,8 @@
 from magma.pipelined.qos.common import QosManager
 from magma.pipelined.service_manager import Tables
 from magma.subscriberdb.sid import SIDUtils
-from scripts.helpers.ng_set_session_msg import CreateSessionUtil
-from scripts.helpers.pg_set_session_msg import CreateMMESessionUtils
```

Comment:
can you move this lib to pipelineD module.

Reply:
Yes, changes done as requested.